lib/peer.js

```diff
@@ -57,6 +74,18 @@ exports.createTCPOutgoingPeer = (addr, swarm) => {
   return peer
 }
 
+/**
+ * Outgoing uTP peers start out with just an IP address. At some point (when there is an
+ * available connection), the client can attempt to connect to the address.
+ */
+exports.createUTPOutgoingPeer = (addr, swarm) => {
+  const peer = new Peer(addr, 'utpOutgoing')
+  peer.addr = addr
+  peer.swarm = swarm
+
+  return peer
+}
+
```

Comment:
seems like you could make these into one function to be more DRY but not important.

Reply:
Added a helper function to improve this 👍